### PR TITLE
clean util *

### PR DESCRIPTION
**Pull request by rokkiter**
clean import util *

### CODE REVIEW

File: model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.LockModeType;
 import jakarta.persistence.TypedQuery;
+
 import java.util.*;
```
Comment (vmuzikar) [inline model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java:39]: Should this be fixed as well? :)

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@rokkiter Thank you for the contribution!

File: model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java (hunk 1 above)
Comment (rokkiter) [reply model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java:39]: If we take this apart, there will be more than five.  
I think it is more reasonable to use * directly in this case.

Comment (vmuzikar) [reply model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java:39]: We have an unwritten rule not to use star imports anywhere in the project. :)

Comment (rokkiter) [reply model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java]: Thanks

**Review by ahus1 — DISMISSED** [review]
Thank you for fixing this code formattings!

**Comment by ahus1** [comment]
@rokkiter - there are some compile errors, please have a look:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project keycloak-core: Compilation failure: Compilation failure: 
Error:  /home/runner/work/keycloak/keycloak/core/src/main/java/org/keycloak/TokenVerifier.java:[195,13] cannot find symbol
Error:    symbol:   class List
Error:    location: class org.keycloak.TokenVerifier<T>
Error:  /home/runner/work/keycloak/keycloak/core/src/main/java/org/keycloak/TokenVerifier.java:[349,39] cannot find symbol
Error:    symbol:   class List
Error:    location: class org.keycloak.TokenVerifier<T>
Error:  /home/runner/work/keycloak/keycloak/core/src/main/java/org/keycloak/TokenVerifier.java:[123,23] cannot find symbol
Error:    symbol:   class List
Error:    location: class org.keycloak.TokenVerifier.TokenTypeCheck
Error:  /home/runner/work/keycloak/keycloak/core/src/main/java/org/keycloak/TokenVerifier.java:[125,31] cannot find symbol
Error:    symbol:   class List
Error:    location: class org.keycloak.TokenVerifier.TokenTypeCheck
```

**Comment by rokkiter** [comment]
Thank, It has been modified

**Review by vmuzikar — DISMISSED** [review]

**Comment by ahus1** [comment]
@rokkiter - there are more compilation errors, for example the one below. Check the logs for other errors.

```
Error:  /home/runner/work/keycloak/keycloak/docs/documentation/tests/src/test/java/org/keycloak/documentation/test/utils/LinkUtils.java:[25,13] cannot find symbol
  symbol:   class Map
  location: class org.keycloak.documentation.test.utils.LinkUtils
```

**Review by vmuzikar — DISMISSED** [review]

**Comment by rokkiter** [comment]
I have been waiting for this PR for a while. Can you reivew it when you have time? Thanks a lot.
https://github.com/keycloak/keycloak/pull/20378

**Review by vmuzikar — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]

**Pull request merged**